codec-http2/src/main/java/io/netty/handler/codec/http2/Http2CodecUtil.java

```diff
@@ -104,12 +104,13 @@
     public static final int DEFAULT_WINDOW_SIZE = 65535;
     public static final short DEFAULT_PRIORITY_WEIGHT = 16;
     public static final int DEFAULT_HEADER_TABLE_SIZE = 4096;
+    public static final long INITIAL_SETTINGS_HEADER_LIST_SIZE = 8192;
```

Comment:
I think this is a different comment and makes sense to allow this value to be configured in the constructors. Note this is the only settings that can be set to a different initial value than what is defined in the RFC.